ghost/email-service/lib/BatchSendingService.js

```diff
@@ -533,6 +537,20 @@ class BatchSendingService {
         const BATCH_SIZE = this.#sendingService.getMaximumRecipients();
         if (mappedMemberLikes.length > BATCH_SIZE) {
             logging.error(`Batch ${batchId} has ${mappedMemberLikes.length} members, but the sending service only supports ${BATCH_SIZE} members per batch.`);
+
+            // @NOTE below is a throwaway code, that's why it is dirty and lives here instead of a nice module
+            if (this.#debugStorageFilePath) {
+                const fs = require('fs-extra');
+                const path = require('path');
+                const currentTimeFilename = new Date().toISOString().replace(/:/g, '-').replace(/\./gi, '-');
+                const outputFileName = `email-batch-sending-members-${currentTimeFilename}.json`;
+                const outputFilePath = path.join(this.#debugStorageFilePath, '/', outputFileName);
+                const jsonModels = models.map(m => m.toJSON());
```

Comment:
This bit might be flaky. I'll change to a raw "JSON.stringify" on the models instance because we don't really know the shape of this object and could expect anything to happen here